tests/Sdk/Acceptance/Extension/Tasks/ESLintTaskCest.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Acceptance\Extension\Tasks;
+
+use PHPUnit\Framework\Assert;
+use SprykerSdk\Sdk\Tests\AcceptanceTester;
+
+class ESLintTaskCest
+{
+    /**
+     * @var string
+     */
+    private const COMMAND = 'validation:frontend:eslint';
+
+    /**
+     * @var string
+     */
+    private const CONFIG_FILE = 'node_modules/@spryker/frontend-config.eslint/.eslintrc.js';
+
+    /**
+     * @param \SprykerSdk\Sdk\Tests\AcceptanceTester $I
+     *
+     * @return void
+     */
+    public function testFilesDontContainViolations(AcceptanceTester $I): void
+    {
+        // Arrange
+        $I->cleanReports();
+
+        // Act
+        $process = $I->runSdkCommand([
+            static::COMMAND,
+            '--file=src/ESLint/success',
+            '--config=' . $I->getPathFromSdkRoot(static::CONFIG_FILE),
+        ]);
+
+        // Assert
+        Assert::assertTrue($process->isSuccessful());
+        Assert::assertFileExists($I->getPathFromProjectRoot('reports/eslint.codestyle.json'));
+        Assert::assertStringNotContainsString('Violations found in files', $process->getOutput());
+    }
+
+    /**
+     * @param \SprykerSdk\Sdk\Tests\AcceptanceTester $I
+     *
+     * @return void
+     */
+    public function testConfigNotFound(AcceptanceTester $I): void
+    {
+        // Arrange
+        $I->cleanReports();
+
+        // Act
+        $process = $I->runSdkCommand([
+            static::COMMAND,
+            '--file=src/ESLint/success',
+            '--config=notExists.js',
+        ]);
+
+        // Assert
+        Assert::assertFalse($process->isSuccessful());
+        Assert::assertFileExists($I->getPathFromProjectRoot('reports/eslint.codestyle.json'));
+        Assert::assertEmpty(file_get_contents($I->getPathFromProjectRoot('reports/eslint.codestyle.json')));
```

Comment:
`'reports/eslint.codestyle.json'` should be constant, you use this value a few times

Reply:
fixed